### PR TITLE
Added in a call to gNMI GetRequest

### PR DESCRIPTION
**Pull request by SeanCondon**

### CODE REVIEW

File: web/onos-gui/src/app/app.component.ts [hunk 1]
```diff
@@ -42,20 +56,68 @@ export class AppComponent {
...
+    this.gnmi.requestGetAllByDevice(deviceId, (e, r) => this.handleGetResponse(e, r));
+  }
+
+  /**
+   * Callback for handling the GetResponse
+   * @param e error
+   * @param r GetResponse
+   */
+  handleGetResponse(e: grpcWeb.Error, r: GetResponse): void {
+    console.log('GetResponse received');
```
Comment (Andrea-Campanella) [inline web/onos-gui/src/app/app.component.ts]: want to add the response here in the log?

File: web/onos-gui/src/app/app.component.ts [hunk 2]
```diff
@@ -42,20 +56,68 @@ export class AppComponent {
...
+    if (e != null) {
+      console.warn('Error getting capabilities from onos-config');
```
Comment (Andrea-Campanella) [inline web/onos-gui/src/app/app.component.ts]: update log

File: web/onos-gui/src/app/app.component.ts [hunk 3]
```diff
@@ -42,20 +56,68 @@ export class AppComponent {
...
+          };
+          this.updates.push(jsonUpdate);
+          console.log('Update', jsonUpdate);
+        }
+      }
+    }
+    console.log('GetResponse finished');
+  }
+
+  pathToString(p: Path): string {
```
Comment (Andrea-Campanella) [inline web/onos-gui/src/app/app.component.ts]: is this method going to end up as complicated as the ones we have in onos-config are are we going to skip dealing with namespaces and keys here ?

Comment (SeanCondon) [reply web/onos-gui/src/app/app.component.ts]: good question - i'm not sure if I'm going to turn them in to paths like we have internally

File: web/onos-gui/src/app/app.component.ts (hunk 1 above)
Comment (SeanCondon) [reply web/onos-gui/src/app/app.component.ts]: will do

File: web/onos-gui/src/app/app.component.ts (hunk 2 above)
Comment (SeanCondon) [reply web/onos-gui/src/app/app.component.ts]: will do

File: web/onos-gui/src/app/app.component.ts (hunk 3 above)
Comment (Andrea-Campanella) [reply web/onos-gui/src/app/app.component.ts]: if there is some son library we could ask for the elements as json values and reuse that. I'd avoid complicating this very much.

Comment (SeanCondon) [reply web/onos-gui/src/app/app.component.ts]: it wasn't being used. will have to add something at some stage in the future

**Pull request merged**